zulipterminal/config/themes.py

```diff
@@ -110,6 +112,8 @@
 GRAY24 = "#928374"  # gray_244
 LIGHTMAGENTA24 = "#b16286"  # neutral_purple
 LIGHTMAGENTABOLD24 = f"{LIGHTMAGENTA24}, bold"
+DARKCYAN24 = "#458588"
+BROWN24 = "#b57614"
```

Comment:
There was no prior comment from where to take these, so perhaps @rht could comment on if these meet expectations?
eg. should darkcyan be aqua rather than neutral_blue? brown is faded_yellow right now.
In any case we should have comments for what these extra colors map to (or come from)

Since we lose these comments in the transfer to the new approach, perhaps it would be useful to rename these colors to match the original names in advance? Hmm, though the conversion script may lose the 16-color interpretation?

Reply:
Yes I was thinking the same thing about using gruvbox names. We do have a 16 color column as well. It has to be manually done.

Re colors, felt these were the closest. I even checked what the color code was called online and it said dark cyan. So, I went with it.

Reply:
What is the source of this 2 new color codes?

Reply:
Is it in the gruvbox vim?

Reply:
Yes, from the same link you have provided.